portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,13 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+async function getComments() {
+  const response = await fetch('/data');
+  const json = await response.json();
+  let comments = json.comments;
+  const commentContainer = document.getElementById('comments');
+  for (const comment of comments) {
+    commentContainer.innerHTML += comment;
```

Comment:
do you need to reset the content? What happens if you click the "Get comments" button twice?

Reply:
I'm using `+=` so it's appending it to the end of the `#comments` element. Pressing the button twice appends the comments twice and you can see both afterwards.

Reply:
It's just a "get comments and display" button right rather than a "show comments" button that fetches the data once and you can toggle them to display/not display. The latter is more useful but I thought I would change it to that at a later time.